build/configs/default_config.yaml

```diff
@@ -99,10 +99,13 @@ accounts:
   keystore: /tmp/accounts
 
 # Interface where the API and P2P Server listens to
-nodeHostname: 0.0.0.0
+nodeHostname: 127.0.0.1
 # Port where API Server listens to
 nodePort: 8082
 
+# Port where GRPC server listens to
+grpcPort: 28202
```

Comment:
why do we need this in the config?
Does a client need to configure the grpc server for any reason? If so its not desirable since we are exposing something that we are going to remove soon.

Reply:
We already were exposing GRPC thriugh HTTP API. That means, we were exposing the GRPC to the public. This PR is changes that and exposes GRPC server on localhost. And we cannot assume the GRPC port we chose to be avaible during the start of the cent-node. So, we will have to expose it here